communication guidelines.md

```diff
@@ -0,0 +1,76 @@
+## Purpose:
+[GitHub Actions](https://docs.github.com/en/actions/learn-github-actions) gives you the flexibility to build an automated software development lifecycle workflow. With [GitHub Actions for Azure](https://docs.microsoft.com/en-gb/azure/developer/github/) you can create workflows that you can set up in your repository to build, test, package, release and deploy to Azure
+This document is to set communication guidelines for making GitHub Actions ready for the external community. We want to set a robust framework for communication, and governance for anything related to GitHib Actions for Azure, for enabling both internal and external users to be always updated on any change in this area.
```

Comment:
Typo for GitHub in the statement "anything related to GitHib Actions"

Reply:
Updated